website/components/i18n/en.json

```diff
@@ -25,7 +25,7 @@
   "webpack-filter-warnings-plugin-desc": "Use [ignoreWarnings](/config/other-options#ignorewarnings) instead",
   "speed-measure-webpack-plugin-desc": "Use [Rsdoctor](/guide/optimization/use-rsdoctor) instead",
   "eslint-webpack-plugin-desc": "Use [eslint-rspack-plugin](https://github.com/rspack-contrib/eslint-rspack-plugin) instead",
-  "circular-dependency-plugin-desc": "Use ESLint [import/no-cycle](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-cycle.md) rule instead",
+  "circular-dependency-plugin-desc": "Use [CircularDependencyRspackPlugin](/plugins/rspack/circular-dependency-rspack-plugin) instead",
```

Comment:
Can we update https://rspack.dev/plugins/#rspack-only-built-in-plugins too

Reply:
done